test/run_tests.sh

```diff
@@ -235,6 +235,7 @@ function run_xla_op_tests3 {
   # NOTE: this line below is testing export and don't care about GPU
   PJRT_DEVICE=CPU CPU_NUM_DEVICES=1 run_coverage "$CDIR/test_core_aten_ops.py"
   run_test "$CDIR/test_pallas.py"
+  run_test "$CDIR/test_megablox.py"
```

Comment:
you want to add this test to https://github.com/pytorch/xla/blob/master/test/tpu/run_tests.sh#L27, and add tag `tpuci` to this pr. Running custom kernel test on CPU and GPU(which is what `test/run_tests.sh` do) will likely fail..